pom.xml

```diff
@@ -786,6 +786,7 @@
           <configuration>
             <source>${java.version}</source>
             <target>${java.version}</target>
+            <forceJavacCompilerUse>true</forceJavacCompilerUse>
```

Comment:
Is there a reason to add it?

Reply:
It's my local problem with maven build. I hope It won't affect other people.